package.json

```
@@ -92,10 +92,15 @@
     "nodemon": "^1.19.4",
     "postcss": "^7.0.27",
     "prettier-eslint": "^9.0.1",
-    "prettier-eslint-cli": "^5.0.0"
+    "prettier-eslint-cli": "^5.0.0",
+    "typescript": "^3.8.3"
   },
   "engines": {
     "node": "12",
     "npm": "6.13"
-  }
+  },
+  "browsers": [
```

Comme indiqué vers le message d'erreur, mettre à jour comme ceci ?
```suggestion
  "browserslist": [
```